### DISCUSSION TITLE
Can I update an old website through command line?

### DISCUSSION THREAD

**Discussion by sr-kai** [datenstrom/community#720] [Ask a question] [answered]
Hi,

I'm trying to upgrade yellow, but I'm getting this error:
```
php yellow.php update
ERROR updating files: Can't download file 'https://github.com/datenstrom/yellow-extensions/raw/master/version.ini'!
Yellow update: Website not updated, 0 updates installed
```
If this helps here is the about command:

```
php yellow.php about
Datenstrom Yellow 0.8.5, PHP 7.2.24-0ubuntu0.18.04.10, CLI Linux
Blog 0.8.4
Bundle 0.8.5 
Command 0.8.6
Contact 0.8.2
Core 0.8.5
Edit 0.8.11
English 0.8.12
Feed 0.8.4
French 0.8.12
German 0.8.12
Image 0.8.2
Markdown 0.8.8
Meta 0.8.8
Paris 0.8.5
Search 0.8.4
Sitemap 0.8.2
Stockholm 0.8.5
Update 0.8.9
ERROR checking updates: Can't download file 'https://github.com/datenstrom/yellow-extensions/raw/master/version.ini'!
```

I've also tried adding different plugins like TOC or Previousnext. I copy the zip file into the extensions folder, and they just disappear without anything being added :( 

I would greatly appreciate any help 🤗 Thanks!

**Comment by markseuffert** [comment, marked as answer]
No, unfortunately the [automatic update](https://github.com/annaesvensson/yellow-update) doesn't work with such an old version.

1. You can transfer your website to a new installation of Datenstrom Yellow. Some assembly required.
2. Alternatively contact the commercial support. You can get your entire website updated for a small fee.

Hope this helps and happy coding. :smile:
